addons/packages/cartographer/0.2.2/README.md

```diff
@@ -0,0 +1,120 @@
+# Package for Cartographer
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Overview](#overview)
+- [Pre-requisites](#pre-requisites)
+- [Installation](#installation)
+- [Documentation](#documentation)
+- [Contributing](#contributing)
+- [License](#license)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview
+
+[carvel]-based Packaging of [Cartographer].
+
+## Pre-requisites
+
+- [kapp-controller]
+- [Tanzu CLI]
+- [cert-manager]
+
+## Installation
+
+First, submit the Package and PackageMetadata objects to the cluster
+
+```bash
+kubectl apply \
+  -f https://github.com/vmware-tanzu/package-for-cartographer/releases/download/v0.0.0/package.yaml
+  -f https://github.com/vmware-tanzu/package-for-cartographer/releases/download/v0.0.0/package-metadata.yaml
+```
+
+With Package and PackageMetadata added to the Kubernetes cluster, proceed with
+installing the package:
+
+```bash
+tanzu package install cartographer \
+  --package-name cartographer.community.tanzu.vmware.com \
+  --version 0.0.0
```

Comment:
feedback on slack:

> _There’s no values that can be customized, it’ll be ideal to also note this fact_

same as the other _todo_, it'd be good to check how other packages report configuration values (or lack thereof) and improve this

Reply:
added a section that mentions that there are no values to be configured.